src/mongo/db/pipeline/document_source_lookup_test.cpp

```diff
@@ -117,9 +117,9 @@ class ReplDocumentSourceLookUpTest : public DocumentSourceLookUpTest {
 };
 
 auto makeLookUpFromBson(BSONElement elem, const boost::intrusive_ptr<ExpressionContext>& expCtx) {
-    auto docSource = DocumentSourceLookUp::createFromBson(elem, expCtx);
-    auto lookup = static_cast<DocumentSourceLookUp*>(docSource.detach());
-    return std::unique_ptr<DocumentSourceLookUp, DocumentSourceDeleter>(lookup,
+    auto docSource = DocumentSourceLookUp::createFromBson(elem, expCtx);              
+    auto lookup = static_cast<DocumentSourceLookUp*>(docSource.detach());                                      
+    return std::unique_ptr<DocumentSourceLookUp, DocumentSourceDeleter>(lookup,                                                                                                         
```

Comment:
could you delete the trailing whitespace that was added. If I understand thisdiff correctly